nodejs/awsx/ecs/container.ts

```diff
@@ -270,6 +270,30 @@ export interface ContainerDependency {
     containerName: pulumi.Input<string>;
 }
 
+/**
+ * See https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_FirelensConfiguration.html
+ * for more details
+ */
+export interface FirelensConfiguration {
+    /**
+     * The options to use when configuring the log router. This field is optional and can be used to specify a custom
+     * configuration file or to add additional metadata, such as the task, task definition, cluster, and container
+     * instance details to the log event.
+     */
+    options?: pulumi.Input<FirelensConfigurationOptions>;
+
+    /**
+     * The log router to use.
+     */
+    type: pulumi.Input<"fluentd" | "fluentbit">;
+}
+
+export interface FirelensConfigurationOptions {
```

Comment:
I don't think this is right.  `options` is a "string to string map", not a strongly typed object.  The typing of this in https://github.com/pulumi/pulumi-aws/blob/d1b5c8a4765ae03efddb580e2d18f45d63d00a29/sdk/nodejs/ecs/container.ts#L73 is "correct" I believe.

Reply:
good catch here! Thanks for that - updated now